src/Illuminate/Foundation/Console/Tinker/Presenters/EloquentCollectionPresenter.php

```diff
@@ -0,0 +1,41 @@
+<?php namespace Illuminate\Foundation\Console\Tinker\Presenters;
+
+use Psy\Presenter\ArrayPresenter;
+use Illuminate\Database\Eloquent\Collection;
+
+class EloquentCollectionPresenter extends ArrayPresenter {
+
+	/**
+	 * EloquentCollectionPresenter can present Collections.
+	 *
+	 * @param  mixed  $value
+	 * @return bool
+	 */
+	public function canPresent($value)
+	{
+		return $value instanceof Collection;
+	}
+
```

Comment:
It is. For some reason my `.editorconfig` plugin isn't working correctly and forgets to change the whitespace settings when opening files :-/